tests/test_views.py

```diff
@@ -1,15 +1,32 @@
 import pytest
-
+from django.urls import reverse
+from collections import OrderedDict
 
 def test_api_parse_succeds(client):
-    # TODO: Finish this test. Send a request to the API and confirm that the
-    # data comes back in the appropriate format.
     address_string = '123 main st chicago il'
-    pytest.fail()
+
+    expected_return_input_string = '123 main st chicago il'
+    expected_return_address_components = OrderedDict([('AddressNumber', '123'), ('StreetName', 'main'), ('StreetNamePostType', 'st'), ('PlaceName', 'chicago'), ('StateName', 'il')])
+    excpected_return_address_type = 'Street Address'
+
+    url = reverse('address-parse')
+    # send address string to API endpoint
+    response = client.get(url, {'input_string': address_string})
+
+    assert response.status_code == 200
+    assert response.data['input_string'] == expected_return_input_string
+    assert response.data['address_components'] == expected_return_address_components
+    assert response.data['address_type'] == excpected_return_address_type
+    # pytest.fail()
```

Comment:
You don't need this commented-out code.

Reply:
👍 (Updated [here](https://github.com/noah-marcus/code-challenge/commit/dfce5546aa28a2b5c209791f2002d23f288d445c))